python/examples/test_xworld.py

```diff
@@ -32,7 +32,7 @@ def compute_speak_action(states, num_actions):
         "task_mode": "arxiv_lang_acquisition",
         "context": 1,
         "pause_screen": True,
-        "task_groups_exclusive": True,
+        "task_groups_exclusive": False,
```

Comment:
No need to introduce this diff?

Reply:
This flag will be automatically set to False later, to replicate the setting of the paper.